src/test/common/test_sloppy_crc_map.cc

```diff
@@ -11,7 +13,7 @@ void dump(const SloppyCRCMap& scm)
   f->open_object_section("map");
   scm.dump(f);
   f->close_section();
-  f->flush(cout);
+  f->flush(std::cout);
```

Comment:
i think, next time, we can just put a `using namespace std` in the front of .cc file to avoid the code churn?